release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -55,6 +55,10 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-4-17-installation-and-update_{context}"]
 === Installation and update
 
+==== Installing a cluster on {aws-full} using an existing IAM profile
+
+With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. You can specify either an existing IAM profile by using the `iamRole` parameter or you can specify a new IAM profile by using the new `iamProfile` parameter, but you cannot specify both. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```

Comment:
I don't think we need to mention `iamRole` in the release note, as we already have a clear description in the [parameter table](https://github.com/openshift/openshift-docs/pull/81715)

```suggestion
With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```